client/README.md

```diff
@@ -62,3 +62,39 @@ To generate a **class** component, run the following
 ```sh
 hygen component-class new --name NameOfComponent
 ```
+
+#### Icons
+##### Usage
+In `App.js` we will add the following and if you would like to add more icons to the library you will just import, add to library, and seperate by commas like below.
+```
+import { library } from ‘@fortawesome/fontawesome-svg-core’
+import { fab } from ‘@fortawesome/free-brands-svg-icons’
+import { fas } from ‘@fortawesome/free-solid-svg-icons’
+import { far } from ‘@fortawesome/free-regular-svg-icons’
+import { faCoffee } from ‘@fortawesome/free-solid-svg-icons’
+library.add(fab, far, fas, faCoffee)
+```
+To use globally, make sure to have the import syntax in the component you would like to use an icon in. To use any of these icons:
+```
+import React from ‘react’
+import { FontAwesomeIcon } from ‘@fortawesome/react-fontawesome’
+export const model = () => (
+  <div>
+    <FontAwesomeIcon icon=“check-square” />
+    Favorite beverage: <FontAwesomeIcon icon=“coffee” />
+    <FontAwesomeIcon icon={[‘fab’, ‘apple’]} />
+    <FontAwesomeIcon icon={[‘fab’, ‘microsoft’]} />
+     <FontAwesomeIcon icon={[‘far’, ‘adress-card’]}
+  </div>
+)
```

Comment:
Can you delete this? Everyone should use your `Icon` component, rather than use the `FontAwesomeIcon` component globally

Reply:
This is how the NPM package is supposed to be used hence my original explanation. But I created the icons in storybook as another means of calling them. But that is how the syntax is supposed to be used with the NPM package.